src/app/src/components/AboutSlide.js

```diff
@@ -110,29 +114,32 @@ export default class AboutSlide extends Component {
                         <FontAwesomeIcon icon={['fa', muteIcon]} />
                     </MuteButton>
                 </VideoContainer>
-                <Flex
-                    width={2 / 5}
-                    padding='2rem'
-                    margin='auto'
-                    flexDirection='column'
-                >
-                    <Heading as='h2' variant='small'>
+                <Box width={2 / 5} pl='3rem'>
```

Comment:
Can we move the above to a named style block?

Reply:
Yep!